app/assets/templates/adminhome.html.erb

```diff
@@ -11,5 +11,9 @@
     <div class="row">
       <%= link_to 'Documents', documents_path, class: 'btn btn-info btn-lg col-md-12' %>
     </div>
+    <br>
+    <div class="row">
+      <%= link_to 'Books', books_path, class: 'btn btn-info btn-lg col-md-12' %>
```

Comment:
Do you want to make it a different color? like using btn-warning (instead of btn-info) to get orange?